internal/cloud/backend_plan.go

```diff
@@ -615,6 +615,13 @@ func (b *Cloud) shouldRenderStructuredRunOutput(run *tfe.Run) (bool, error) {
 	if b.client.IsEnterprise() {
 		tfeVersion := b.client.RemoteTFEVersion()
 		if tfeVersion != "" {
+			semVersion := tfeSemVersion(tfeVersion)
```

Comment:
❓ Rather than extending a logic that only affects customers on a release < 202302, we might as well just remove it.

We are currently in 202507, that is 2 years and 5 months later.

How does the @hashicorp/terraform-core team think about deprecation? 

see
```
if releaseDate < 202302 {
```

Reply:
I'd be happy to defer to the core-cloud team on this - if they think it's okay then I'd be happy with that. What's the real impact on that? Someone using such an old version of TFE wouldn't be able to use latest versions of Terraform with it?